src/components/input/bl-input.ts

```diff
@@ -179,14 +179,16 @@ export default class BlInput extends LitElement {
       : '';
     const label = this.label ? html`<label>${this.label}</label>` : '';
 
+    const classes = {
```

Comment:
We can write it like this to make this cleaner. Also, for consistentcy with the button

```js
const classes = classMap({
      'dirty': this.dirty,
      'has-icon': this.icon || (this.dirty && this._invalidState),
      'has-value': this.hasValue,
});
```